tests/unit/drivers/crc/main.c

```diff
@@ -30,11 +31,51 @@ void test_crc16_ansi(void)
 	zassert(crc16_ansi(test2, sizeof(test2)) == 0x9ecf, "pass", "fail");
 }
 
+void test_crc_ccitt(void)
```

Comment:
keep the 16 in crc16.

Reply:
Will do based on the answer to the previous comment.